plugins/presentation/src/components/AttributeEditor.svelte

```diff
@@ -19,6 +19,7 @@
 
   export let value: unknown
   export let attribute: AttrModel
+  export let editable = true
```

Comment:
It should be always editable :-) Why we need ```editable``` for AttributeEditor?

Reply:
Because we do not have an understanding of whether we should separate editing and viewing attributes on different elements, and if so, how to do it. Previously, the "editable" was not passed to the child elements, and they took the default value, which they had different (for example, simple text editing was read-only), I decided that this is the most flexible solution in conditions of uncertainty

Reply:
Nope, it will just increase burned, Please pass editable to child elements as true, since AttributeEditor should be used only with edit, I've not see a case why AttributeEditor should be used with editable = false